configtx/config_test.go

```diff
@@ -19,6 +19,32 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func TestNewConfigTx(t *testing.T) {
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+
+	original := &cb.Config{
+		ChannelGroup: &cb.ConfigGroup{
+			Version: 7,
+			Values: map[string]*cb.ConfigValue{
+				"foo": {
+					Version: 3,
+					Value:   []byte("value1value"),
+				},
+				"bar": {
+					Version: 6,
+					Value:   []byte("value2value"),
+				},
+			},
+		},
+	}
+
+	c := New(original)
+	gt.Expect(proto.Equal(c.OriginalConfig(), original)).To(BeTrue())
+	gt.Expect(proto.Equal(c.UpdatedConfig(), original)).To(BeTrue())
```

Comment:
Maybe apply a very simple update to the original for this check?

Reply:
Done